src/govuk/helpers/spacing.test.js

```diff
@@ -66,6 +82,36 @@ describe('@function govuk-spacing', () => {
         'Expected a number (integer), but got a string.'
       )
   })
+
+  it('throws an error when passed a non-existent point', async () => {
```

Comment:
Do you think it's worth adding a test for being passed a non-existent negative point?

Reply:
Good suggestion, thanks! 👍🏻